dll/win32/msgina/gui.c

```diff
@@ -12,6 +12,8 @@
 #include <winnls.h>
 #include <winreg.h>
 
+#define IDT_BAR 1
```

Comment:
`IDT_BAR` should probably go to resource header file.

Reply:
It can. Done.